docs/src/getting_started.md

```diff
@@ -16,7 +16,7 @@ Introducing the diffusion flux $q$, we can rewrite equation `(1)` as a system of
 
 ```math
 \begin{align}
-\boldsymbol{q}                &= -\chi \nabla C~, \\
+\boldsymbol{q}                &= -\chi \nabla C~, \\[5pt]
```

Comment:
I've added some spacing between the equations. There seems to be some rendering issues with the equation number in the margin (independently of adding spacing or not) in Safari.

<img width="496" alt="Screenshot 2024-07-23 at 21 49 50" src="https://github.com/user-attachments/assets/b2622be1-089c-40ce-b0ed-1546d30bd7db">

Reply:
The issue seems to be the `align` environment. I don't know if this is a bug in LaTex rendering or Documenter, reverting for now to multiple equations